android/app/src/main/java/org/mozilla/magnet/ScannerBle.java

```diff
@@ -66,6 +66,11 @@ public void start() {
     @ReactMethod
     public void stop() {
         Log.d(TAG, "stop");
+        if (bluetoothAdapter == null || !bluetoothAdapter.isEnabled()) {
+            // Device does not support bluetooth, or is not enabled
+            return;
+        }
+
```

Comment:
Hmm, we'd still need to check for `isEnabled` in both places regardless.  Not certain it's entirely worth duplicating the state of `bluetoothAdapter`.